src/main/java/com/asgarov/Main.java

```diff
@@ -0,0 +1,13 @@
+package com.asgarov;
+
+public class Main {
+
+    public static void main(String[] args) {
+//        for (int i = 0; i < 10; i++) {
+//            new Thread(() -> {
+//                ChairWarehouse.getInstance();
+//                ChairWarehouse.getInstance().sellChairFromStock(1);
+//            }).start();
+//        }
```

Comment:
clean code tip from R. Martin - never commit commented lines unless they have a purpose (javadoc/explanation of the code)